huxley/www/static/js/huxley/components/RegistrationView.js

```diff
@@ -571,6 +541,15 @@ var RegistrationView = React.createClass({
     this.setState(change);
   },
 
+   _handleCommitteePreferenceChange: function(committee) {
+    var index = this.state.committee_prefs.indexOf(committee.id);
+    if (index < 0) {
+      this.state.committee_prefs.push(committee.id);
```

Comment:
You cannot modify `this.state` directly. Changes to state have to happen through `setState`.
This, for example, would be
```js
this.setState({
  committee_prefs: this.state.committee_prefs.concat(committee.id),
});
```
We use `concat()` so that a new array is created and the old one isn't modified.